src/spatial-grid.js

```diff
@@ -134,10 +133,13 @@ export const spatial_grid = (() => {
       for (let x = i1[0], xn = i2[0]; x <= xn; ++x) {
         for (let y = i1[1], yn = i2[1]; y <= yn; ++y) {
           const k = this._Key(x, y);
-          if (!(k in this._cells)) {
-            this._cells[k] = new Set();
-          }
-          this._cells[k].add(client);
+          const cellClientsCheck = this._cells.get(k);
+          // NOTE: calling .get() on a Map is apparently currently (Sept 2022)

```

Comment:
Looks like this isn't the case anymore?
https://chromium-review.googlesource.com/c/v8/v8/+/3709246